atlp-rwanda/rca-phantom-team1-bn
# Feat/routes crud

## What does this PR do

-   It shows implementation of Routes model, service, controller and routes.

## Description

- As  **an operator**, I want to create routes so that buses can travel in pre-defined routes.

- As  **an operator**, I want to create routes so that users can view in pre-defined routes.

## How to manually test this

-   Check in the branch  `feat/routes-crud`
-   Run the server
-   Open the Swagger API documentation to test the buses

## Relevant Trello story

[7-crud-operations-for-routes][(https://trello.com/c/siqznvua/7-crud-operations-for-routes)](https://trello.com/c/siqznvua/7-crud-operations-for-routes)

## Screenshots

![enter image description here](https://res.cloudinary.com/dpqasrwfu/image/upload/v1679992665/Screen_Shot_2023-03-28_at_10.35.11_zqoawu.png)
